### PR TITLE
Add plugin title to admin notice

### PR DESCRIPTION
**Pull request by RichardCoffee**
This adds the plugin title to the admin notice displayed as the result of a manual update check.

Just a small tweak to provide an end user with more information about what is occurring.

### CODE REVIEW

**Comment by YahnisElsts** [comment]
Looks good. Just one thing: WordPress docs say not to call the `translate()` function directly. Is there are particular reason it's used here? If not, it should probably be replaced with `__()`.

**Comment by RichardCoffee** [comment]
Yes, I saw that in the docs also.  While preferable, using any of the normal WordPress translation functions properly requires that both the string and the text domain be literals, neither of which are possible in this case.  Since the plugin name and text domain here are both variables, it seemed to me that the best way to go was to use WordPress' internal translate() function.  From what I have read on the subject, WordPress takes the stance they do because they want to discourage possible misuse of the function, something with which I would agree.  Using __() might give a result, but I think that would be a misuse of _that_ function. :)

Note:  If WordPress's makepot.php has been used to create the .pot file, then the translation can be expected to be present.  However, if Poedit was used, then it is possible that the string will not have been translated, if it only appears in the plugin header.  This is because makepot adds the header strings to the .pot file and Poedit does not.

So basically, I just used translate() because I thought it was better to under the circumstances.

**Comment by RichardCoffee** [comment]
Oops, close the request accidently.  Clicked the wrong button.  :(
